examples/snap-plugin-collector-rand/rand/rand.go

```diff
@@ -54,8 +55,20 @@ var (
 	}
 )
 
+var req bool = false
```

Comment:
@kjlyon, do you mind adding a note in the README that --require-config flag only exists in this rand collector plugin for not altering the current collector behavior but assisting testing.